sources/osg/Notify.js

```diff
@@ -48,14 +48,22 @@ function getStackTrace( err ) {
     return callstack;
 }
 
+var logCache = new window.Map();
```

Comment:
I dont want to have this generalized, It's specific when we want to warn the user only once. So It should be implemented on specific case.
eg you load a file and print a warning, you load a second file you will not have because of this.